workflow/example/case-list.csv

```diff
@@ -0,0 +1,192 @@
+fullname,id,location,size,center_lat,center_lon,top_left_lat,top_left_lon,lower_right_lat,lower_right_lon,left_x,right_x,lower_y,top_y,startdate,enddate
+998-beaufort_sea-1200km-20060504-20060506,998,beaufort_sea,1200km,,,-152.9892,67.6236,-114.8220,78.9424,-2334051,-1127689,-414418,757892,2006-05-04,2006-05-06
```

Comment:
The 998 and 999 cases at the top here are missing the centroid data, we can just delete them. I'm not sure why the ended up here anyway, it was supposed to be a list of the 100 km cases.

Reply:
Done.